x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -26799,6 +26793,12 @@
     "xpack.reporting.uiSettings.validate.customLogo.badFile": "Désolé, ce fichier ne convient pas. Veuillez essayer un autre fichier image.",
     "xpack.reporting.uiSettings.validate.customLogo.tooLarge": "Désolé, ce fichier est trop volumineux. Le fichier image doit être inférieur à 200 kilo-octets.",
     "xpack.reporting.userAccessError.learnMoreLink": "En savoir plus",
+    "generateCsv.esErrorMessage": "Réponse {statusCode} reçue d'Elasticsearch : {message}",
```

Comment:
I don't think these changes should be in this PR, guessing they came in with a merge or rebase?

Reply:
Yeah came with a pull upstream, I tried to remove translations in latest commit, but was not successful.
So did a reset head, which also cleaned up my history. Hope this isn't confusing to review.